src/ResourceManagement/MachineLearning/Microsoft.Azure.Management.MachineLearning/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.10.0-preview",
+  "version": "1.0.0",
```

Comment:
@NonStatic2014 this should be version 1.0.0-preview (or 0.11.0-preview) since you are introducing breaking changes